src/jinja2/ext.py

```diff
@@ -456,11 +457,25 @@ def parse(self, parser):
 
 
 class WithExtension(Extension):
-    pass
+    def __init__(self, environment):
+        Extension.__init__(self, environment)
+        warnings.warn(
+            "WithExtension is deprecated and will be removed in version 3.1",
```

Comment:
The style is to put code names in quotes, and to end with a period. If someone is using this, they're probably referring to it as `jinja2.ext.with_`. `The 'with_' extension is deprecated and will be removed in version 3.1. This is built-in now.`